test/controllers/organizations/staff/default_pet_tasks_controller_test.rb

```diff
@@ -41,6 +41,34 @@ class Organizations::Staff::DefaultPetTasksControllerTest < ActionDispatch::Inte
           post staff_default_pet_tasks_url, params: @params
         end
       end
+
+      should "assign to dog" do
+        assert_authorized_to(
+          :manage?, DefaultPetTask,
+          context: {organization: @organization},
+          with: Organizations::DefaultPetTaskPolicy
+        ) do
+          post staff_default_pet_tasks_url, params: {
+            default_pet_task: {name: "New Dog Task", species: "Dog", due_in_days: 5}
+          }
+          has_dog_species = DefaultPetTask.find_by_name("New Dog Task").Dog?
+          assert_equal true, has_dog_species
```

Comment:
Ah so this is testing that it creates a new DefaultPetTask - but what I meant is that it creates a new `Task`
If you look in app/services/organizations we have the `DefaultPetTaskService` which gets called when we create a new pet, and adds a Task for each DefaultPetTask for that species. 

So, to test, we want to 
- create the default pet task for dog
- create a dog
- verify that dog has a **task** with the same name as the default pet task for dogs
- do the same for cat

does that make sense?

Reply:
bump :)